### DISCUSSION TITLE
Theming `<Spinner />` size prop.

### DISCUSSION THREAD

**Discussion by b-novikov-ipersonality** [chakra-ui/chakra-ui#5465] [Q&A]
Hello there. 

Is the only way to make `<Spinner />` bigger is to set `w` and `h` props? Cannot I customize `size` prop in some way?

**Comment by nikolovlazar** [comment]
Hey @b-novikov-ipersonality! You can also use the `boxSize` prop. Here's a codesandbox with different ways of sizing the Spinner component: https://codesandbox.io/s/spinner-sizing-ktyz3
